### ISSUE TITLE
Biber panics when processing consecutive ands in the author field

### ISSUE THREAD

**Issue by DrGo** [plk/biber#283]
Hello,
Biber is failing with 
`Can't call method "get_field" on an undefined value at /var/folders/kz/fc7nstbx16q2gvb1w6rry8cm0000gn/T/par-73616c6168/cache-6768eefc5099226dccbd9174e351641d1c43cd6b/inc/lib/Biber.pm line 4451.`

when processing this entry

```
@misc {ck56,
title= {test},
howpublished= {hello},
address= {Canada},
address= {Mississauga},
year= {2015},
author= {Raifu A and Franco EL and and the Congo Screening Study Group},
}
```
The issue seems to be consecutive **_and_** in the author field which I inadvertently output from a script that  generates a bib file automatically from a source that uses commas to separate authors names.  

`biber --tool -V -D t.bib  > trace.log`
seems to WARN about the issue, but still fails.
```
substring 3, warning: empty substring
Can't call method "get_field" on an undefined value at /var/folders/kz/fc7nstbx16q2gvb1w6rry8cm0000gn/T/par-73616c6168/cache-6768eefc5099226dccbd9174e351641d1c43cd6b/inc/lib/Biber.pm line 4451.
```
contents of trace.log

> INFO - This is Biber 2.13 running in TOOL mode
> INFO - Logfile is 't.bib.blg'
> INFO - DEBUG mode: all messages are logged to 't.bib.blg'
> INFO - Globbing data source 't.bib'
> INFO - Globbed data source 't.bib' to t.bib
> INFO - Looking for bibtex format file 't.bib'
> INFO - LaTeX decoding ...
> INFO - Found BibTeX data source 't.bib'
> WARN - Name in key 'ck56' is empty (probably consecutive 'and'): skipping entry 'ck56'

Thanks,

**Comment by plk** [comment]
This was a bug. Fixed in 2.14 DEV (now on SourceForge) with regression test in place. Please test and confirm.

**Comment by DrGo** [comment]
Thanks for the prompt reply. Much appreciated.
Sadly I am unable to run it because it is notarized. Another reason to downgrade from Catalina!
https://stackoverflow.com/questions/57466895/can-t-be-opened-because-it-s-integrity-cannot-be-verified-on-osx-catalina

**Comment by DrGo** [comment]
OK. Managed to run it. Th DEV binary is still returning the same error in my testing. But, It is still saying version 2.13. Is that expected?

**Comment by plk** [comment]
No, it should say 2.14 (beta). Did you get it from the development folder on SF?

**Comment by DrGo** [comment]
yes, tried twice to be sure.

**Comment by plk** [comment]
Are you sure that your path is configured to pick up the dev version? Are you replacing the currently installed version or installing it in parallel?

**Comment by DrGo** [comment]
yes, I renamed the old executable and copied the new one into the same folder. I have also run it directly from the downloaded folder.

**Comment by plk** [comment]
I just fetched: https://sourceforge.net/projects/biblatex-biber/files/biblatex-biber/development/binaries/OSX_Intel/biber-darwin_x86_64.tar.gz/download

unpacked to Desktop and ran:

```
> ./biber --version
biber version: 2.14 (beta)
```

Do you get something different?

**Comment by plk** [comment]
The OSX 10.12+ binary for biber is now codesigned (starting with the current 2.14 DEV) and so should, I think, work with Catalina if you can try it ...

**Comment by DrGo** [comment]
Great!
I can confirm that bug is fixed in v2.14.
Thanks,
